### ISSUE TITLE
✨ Improve experience of trying to open unsupported filetype in napari

### ISSUE THREAD

**Issue by liaprins-czi** [napari/napari#3306]
### Overview of design need
- **Is there an existing GitHub issue this design work pertains to?** 
[If image fails to open we should show a visible error in the viewer #1301](https://github.com/napari/napari/issues/1301)
[Feature: Static plugin "manifest" file #2975](https://github.com/napari/napari/issues/2975)

- **Also link to any specific comments or threads where the problem to be solved by design is mentioned**
[Comment with possible suggestions](https://github.com/napari/napari/issues/1301#issuecomment-909486083)

- **In a sentence or two, describe the problem to be solved for users**
When a user attempts to open a file (either by dragging and dropping or via File --> Open (not sure about via console... find out!)) that napari core nor any of the users' already-installed plugins support, they get an error message. Ideally we could help them out by suggesting plugin that can open it (if one exists).

### What level of design is needed? (Choose all that apply)
- [ ] **User experience research:**  high-level recommendation/exploration of user needs, design heuristics, and / or best practices to inform a design experience  
(Use this option when you feel there’s a challenge to be solved, but you’re curious about what the experience should be — may involve research studies to understand challenges/opportunities for design)
- [X] **Information flow / conceptual:** organizing and structuring of information flow and content, including layout on screen or across multiple steps
- [X] **Visual:** creating mockups, icons, etc
(If choosing this level alone, it means that the content to be mocked up and its organization is already known and specified)

### Is design a blocker?
- [ ] **Yes:** engineering cannot proceed without a design first
- [X] **No:** engineering can create a first version, and design can come in later to iterate and refine

If selecting **Yes**, how much design input is needed to unblock engineering? For example, is a full, final visual design needed, or just a recommendation of which conceptual direction to go?

### T-shirt size
- [ ] ●○○○○ XS: able to be done in a typical workday or two
- [ ] ●●○○○ S: able to do within one sprint, if also working on other things in the sprint
- [X] ●●●○○ M: able to do within one sprint, if it is the sole focus
- [ ] ●●●●○ L: will likely take more than one sprint to complete, but less than two
- [ ] ●●●●● XL: a very large experience to be determined or overhauled, will need to be broken down further

**Comment by tlambert03** [comment]
> Ideally we could help them out by suggesting plugin that can open it (if one exists).

yeah, this is a long term goal... this will require A) a _static_ way for a plugin to declare what it can open (we don't have that yet, but see #2975), and B) a database of manifests (including uninstalled ones) that can be queried... like the hub

**Comment by DragaDoncila** [comment]
I believe this issue is a duplicate of #4614 so I'm going to close it.
